### PR TITLE
Core functionalities added

### PR DESCRIPTION
**Pull request by pranavpinarayi**
@nishintn  @aswathym 
Hi,

Please find the pull request for the branch 'CoreFunctionalities' which includes features like adding, saving, and deleting reminders within a new UI.
Hosted  URL: http://hrishikeshsoman.github.io/ReminderApp/

Thanks.

### CODE REVIEW

**Comment by nishintn** [comment]
@pranavpinarayi @hrishikeshsoman ,
Add hand cursor to all buttons.

File: js/uimanipulation.js [hunk 1]
```diff
@@ -1,55 +1,90 @@
...
+    else {
+        show = 0;
+        document.getElementById("containerFinishedDiv").style.display = "block";
+        toggleButton.innerHTML = "Hide completed";
+    }
 }
-
 function addObject(event) {
      if(!event||event.id=="add"||event.keyCode == 13) {
             var contentDiv = document.createElement('Div');
```
Comment (nishintn) [inline js/uimanipulation.js]: Suggest to create a template and load it here. Check with Diaz. He has done it before.

File: js/uimanipulation.js [hunk 2]
```diff
@@ -1,55 +1,90 @@
...
-            buttonDiv.className = "buttonDiv";
-            contentDiv.appendChild(buttonDiv);
+            var completedButton = document.createElement('Div');
+            completedButton.className = "completedButtonDiv";
+            completedButton.setAttribute("onclick","{new reminder(this.nextSibling.nextSibling.firstChild).finishedToggle();}");
+            contentDiv.appendChild(completedButton);
+
+            var deleteButton = document.createElement('Div');
+            deleteButton.className = "deleteButtonDiv";
+            deleteButton.innerHTML="&#10007";
```
Comment (nishintn) [inline js/uimanipulation.js]: ???

File: js/uimanipulation.js [hunk 3]
```diff
@@ -1,55 +1,90 @@
...
-            contentDiv.appendChild(completedButton);
 
-            var deleteButton = document.createElement('Div');
-            deleteButton.className = "deleteButtonDiv";
-            deleteButton.innerHTML="-";
-            deleteButton.setAttribute("onclick","removeObject(this)");
-            contentDiv.appendChild(deleteButton);
+            if(!remArray[index]){
+                document.getElementById('container').appendChild(contentDiv);
+                completedButton.innerHTML = "&#x2610";
```
Comment (nishintn) [inline js/uimanipulation.js]: ???

File: js/uimanipulation.js [hunk 4]
```diff
@@ -1,55 +1,90 @@
...
+              textField.focus();
+              textField.setAttribute("id",idIndex);
+              idIndex++;
+            }
+            else {
+                textField.id = remArray[index].id;
+                textField.value = remArray[index].text;
+                if(remArray[index].complete == "no"){
+                    completedButton.innerHTML = "&#x2610";
```
Comment (nishintn) [inline js/uimanipulation.js]: ??

**Comment by hrishikeshsoman** [comment]
Hi,

I have made the suggested modifications.
Please find the app on : http://hrishikeshsoman.github.io/ReminderApp

Thanks.

**Pull request merged**